src/client/initClient.jsx

```diff
@@ -37,7 +37,8 @@ export default async function initClient() {
 
     await loadPrerenderScripts(store.getState());
 
-    const { redirectLocation, renderProps } = await matchPromise({ history, routes });
+    const asd = await matchPromise({ history, routes });
+    const { redirectLocation, renderProps } = asd;
```

Comment:
Can this be collapsed back into a one liner? or a more detailed variable name chose.

Reply:
my bad, this is from testing, will remove `asd`